### PR TITLE
Changed PDF proposal to correct directory

### PR DESCRIPTION
**Pull request by edoomm**

### CODE REVIEW

**Comment by edoomm** [comment]
@Therealsamumanz check this out and approve it (if you don't see anything wrong with it), once you do that it will be merged automatically to main.

I moved the file to `docs/proposal` as I told you to do it and renamed it to lowercase, this is better since `git` is `UNIX` based and it's case sensitive and you get used to this consistency while naming files.

See: https://developers.google.com/style/filenames

**Comment by Therealsamumanz** [comment]
I've reviewed the proposal and agree with the changes. Will name the files with lowercase from now on.

**Review by Therealsamumanz — APPROVED** [review]
Looks good to me

**Pull request merged**
